code/WorkInProgress/blueprints.dm

```diff
@@ -19,22 +32,31 @@
 /obj/machinery/abcu
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "builder"
-	name = "Automated Blueprint Construction Unit (ABC-U)"
-	desc = "This fine piece of machinery can construct entire rooms from blueprints."
+	name = "\improper ABC Unit"
+	desc = "An Automated Blueprint Construction Unit. \
+		This fine piece of machinery can construct entire rooms from blueprints."
 	density = 1
 	opacity = 0
 	anchored = UNANCHORED
+	processing_tier = PROCESSING_FULL
 
 	var/invalidCount = 0
+	var/building = FALSE
+	var/BuildIndex = 1
+	var/BuildEnd = 0
+	var/list/markers = list()
+	var/list/apclist = list()
+	var/MetalOwed = 0
+	var/CrystalOwed = 0
+	var/TileCostProcessed = FALSE
 
 	var/obj/item/blueprint/currentBp = null
-	var/locked = 0
-	var/building = 0
-
+	var/locked = FALSE
+	var/Paused = FALSE
```

Comment:
Would appreciate some consistency in casing- don't have to change existing vars, but why is `Paused` capitalized while `building` and `markers` aren't? Would prefer to either lowercase or uppercase one-word var names and keep it that way

Reply:
Sorry, somewhere in the middle of this, I decided I didn't like every var and proc and keyword being lowercase. So new stuff I added from then was: vars in title case, procs in camel case. Didn't modify any existing stuff like building or markers or locked.

Reply:
our code style is to use all lowercase and underscores, generally

Reply:
ok, it pained me, but names within /obj/machinery/abcu are standardized now. i still think capitals are practical for readability :p